src/data/schema/10_student_course_selection.sql

```diff
@@ -0,0 +1,21 @@
+DROP TABLE IF EXISTS student_course_selection;
+CREATE TABLE student_course_selection
+(
+	user_id		int,
+	semester	varchar(255),
+	course_id	varchar(255),
+
+	PRIMARY KEY (user_id, semester, course_id),
+	FOREIGN KEY (user_id) REFERENCES user_account(user_id)
+);
+
+INSERT INTO student_course_selection (user_id, semester, course_id) 
+VALUES (
+    (SELECT DISTINCT
+        user_id
+    FROM
+        user_account), 
+    null,
+    null
+);
```

Comment:
ummm, you can't do this. You're mixing a relation and a singular value with this syntax. You either provide a relation or a series of values corresponding to the table's columns.

So change it to 

```
INSERT INTO student_course_selection (user_id, semester, course_id) 
(
    SELECT DISTINCT
        user_id,
        NULL,
        NULL
    FROM
        user_account
);

Reply:
Will do, wasn't sure how this would work

Reply:
For consistency, we decided this block should be removed as there is no reason for initialization and NON-NULL constraint as part of PK will make this impossible anyway.